modules/ROOT/pages/xml-sdk.adoc

```diff
@@ -1211,6 +1211,78 @@ To use the operations from the <<example2, example>> above in a Mule app, it is
 </mule>
 ----
 
+[[xml-sdk-tls]]
+== TLS Support
+
+Starting with Mule 4.6, XML SDK supports Transport Layer Security (TLS). Use TLS to enhance security for your app. XML SDK connectors can support the `<tls:context>` parameter in their configuration parameters.
+
+=== Parameter Declaration
+
+Use the `xmlns:tlsEnabled=true` annotation to indicate the component to support TLS. The annotated element does not necessarily have to be a top level element. In the following example, the annotated element is `<http:request-connection>` which is a child of the `<http:request-config>` top level element.
+
+[source,xml,linenums]
+----
+<http:request-config
+name="concur-httpreq-config" xmlns:connection="true"
+basePath ....>
+  <http:request-connection xmlns:tlsEnabled="true"
+  host=...>
+----
+
+The `<tls:context>` parameter can be specified only in one component. Using it on more than one component results in an error. 
+
+XML SDK adds a `<tls:context>` parameter in the configuration of the generated extension. This parameter is optional unless the `<tls:context>` parameter in the target element is required and doesn’t have a default value.
+
+When the extension is used, the supplied `<tls:context>` parameter is mapped to the config element with this annotation. 
+
+==== Inline Declaration
+
+The following example shows an inline declaration of the `<tls:context>` parameter:
+
+[source,xml,linenums]
+----
+<concur:config name="inline">
+   <tls:context>
+      <tls:key-store keyPassword="changeit"
+         password="changeit"
+         path="ssltest-keystore.jks"/>
+  </tls:context>
+</concur:config>
+----
+
+==== By-Reference Declaration
+
+The following example shows a by-reference declaration of the `<tls:context>` parameter:
+
+[source,xml,linenums]
+----
+<tls:context name="myTlsContext">
+  <tls:key-store keyPassword="changeit"
+    password="changeit"
+    path="ssltest-keystore.jks"/>
+</tls:context>
+<concur:config name="byRef" tlsContext="myTlsContext" />
+----
+
+=== Parameter Default Value
+
+The target element can already provide a value for the `<tls:context>` parameter, for example:
+
+[source,xml,linenums]
+----
+<http:request-config
+name="concur-httpreq-config" xmlns:connection="true"
+basePath ....>
+  <http:request-connection xmlns:tlsEnabled="true"
+  host=...>
+	  <tls:context>
+      <tls:trust-store path="ssltest-cacerts.jks" password="changeit2"/>
+    </tls:context>a
+  </http:request-connection>
+----
+
+In this case, the value that comes from the module definition is used as the default. The synthetic parameter is still added to the module’s config, which, if provided, is used as an override. Providing a default value at the target element makes the synthetic parameter optional even if the parameter at the target element is required. This applies to both inline and by-reference declarations.
```

Comment:
still -> also?

Reply:
"This" 
https://confluence.internal.salesforce.com/display/MTDT/MuleSoft+CX+Writing+Style+Reference#MuleSoftCXWritingStyleReference-%22it,%22%22this,%22%22these%22